### PR TITLE
DOCAPI-7090: SYSTEM DISTRIBUTED docs. EN review. RU translation.

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation for #4935.

### CODE REVIEW

File: docs/en/query_language/system.md [hunk 1]
```diff
@@ -4,31 +4,36 @@
...
 
 ```
 SYSTEM STOP DISTRIBUTED SENDS [db.]<distributed_table_name>
 ```
 
+
 ### FLUSH DISTRIBUTED {#query_language-system-flush-distributed}
 
-Forces ClickHouse to send data to the servers of the cluster in synchronous mode. If some of the servers are not available, ClickHouse throws an exception and stops query processing. When servers are back into operation, you should repeat the query.
+Forces ClickHouse to send data to cluster servers synchronously. If any servers are unavailable, ClickHouse throws an exception and stops processing of the query. You should repeat the query when all servers are back online.
```
Comment (blinkov) [inline docs/en/query_language/system.md]:
```suggestion
Forces ClickHouse to send data to cluster servers synchronously. If any servers are unavailable, ClickHouse throws an exception and stops query execution. You can retry the query until it succeeds, which will happen when all servers are back online.
```

File: docs/ru/query_language/system.md [hunk 2]
```diff
@@ -1,33 +1,36 @@
-# SYSTEM Queries {#query_language-system}
+# Запросы SYSTEM {#query_language-system}
 
 - [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends)
 - [FLUSH DISTRIBUTED](#query_language-system-flush-distributed)
 - [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends)
 
-## Managing Distributed Tables {#query_language-system-distributed}
+## Управление распределенными таблицами {#query_language-system-distributed}
```
Comment (blinkov) [inline docs/ru/query_language/system.md]: одно и то же слово по-разному написано в плане е/ё на соседних строках

File: docs/ru/query_language/system.md [hunk 3]
```diff
@@ -1,33 +1,36 @@
...
-ClickHouse can manage [distributed](../operations/table_engines/distributed.md) tables. When a user inserts data into such table, ClickHouse creates a queue of the data which should be sent to servers of the cluster, then asynchronously sends them. You can control the processing of queue by using the requests [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends), [FLUSH DISTRIBUTED](#query_language-system-flush-distributed) and [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends).
+ClickHouse может оперировать [распределёнными](../operations/table_engines/distributed.md) таблицами. Когда пользователь вставляет данные в эти таблицы, ClickHouse сначала создает очередь данных, которые должны быть отправлены на серверы кластера, а затем асинхронно отправляет подготовленные данные. Вы пожете управлять очередью с помощью запросов [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends), [FLUSH DISTRIBUTED](#query_language-system-flush-distributed) и [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends). Также есть возможность синхронно вставлять распределенные данные с помощью настройки `insert_distributed_sync`.
```
Comment (blinkov) [inline docs/ru/query_language/system.md]:
```suggestion
ClickHouse может оперировать [распределёнными](../operations/table_engines/distributed.md) таблицами. Когда пользователь вставляет данные в эти таблицы, ClickHouse сначала формирует очередь из данных, которые должны быть отправлены на серверы кластера, а затем асинхронно отправляет подготовленные данные. Вы пожете управлять очередью с помощью запросов [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends), [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends) и [FLUSH DISTRIBUTED](#query_language-system-flush-distributed). Также есть возможность синхронно вставлять распределенные данные с помощью настройки `insert_distributed_sync`.
```

insert_distributed_sync — ссылка?

File: docs/ru/query_language/system.md [hunk 4]
```diff
@@ -1,33 +1,36 @@
...
+Отключает фоновую отправку при вставке данных в распределенные таблицы.
 
 ```
 SYSTEM STOP DISTRIBUTED SENDS [db.]<distributed_table_name>
 ```
 
 ### FLUSH DISTRIBUTED {#query_language-system-flush-distributed}
 
-Forces ClickHouse to send data to the servers of the cluster in synchronous mode. If some of the servers are not available, ClickHouse throws an exception and stops query processing. When servers back into operation, you should repeat the query.
+Принудительно активирует разовую синхронную отправку данных на серверы кластера. Если какие-либо серверы недоступны, ClickHouse генерирует исключение и останавливает обработку запроса. Необходимо повторить запрос после восстановления работоспособности серверов кластера.
```
Comment (blinkov) [inline docs/ru/query_language/system.md]:
```suggestion
В синхронном режиме отправляет все данные на серверы кластера. Если какие-либо серверы недоступны, ClickHouse генерирует исключение и останавливает выполнение запроса. Такой запрос можно повторять до успешного завершения, что будет означать возвращение связанности с остальными серверами кластера.
```

кстати, в таком контексте имхо лучше звучат «узлы» вместо «серверов»

File: docs/en/query_language/system.md [hunk 5]
```diff
@@ -4,31 +4,36 @@
...
+ClickHouse can manage [distributed](../operations/table_engines/distributed.md) tables. When a user inserts data into these tables, ClickHouse first creates a queue of the data that should be sent to cluster servers, then asynchronously sends it. You can manage queue processing with the [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends), [FLUSH DISTRIBUTED](#query_language-system-flush-distributed), and [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends) queries. You can also synchronously insert distributed data with the `insert_distributed_sync` setting.
 
 
 ### STOP DISTRIBUTED SENDS {#query_language-system-stop-distributed-sends}
 
-Disables background data distributing, when inserting data into the distributed tables.
+Disables background data distribution when inserting data into distributed tables.
```
Comment (blinkov) [inline docs/en/query_language/system.md]: it's worth to mention what happens if they have already been stopped

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/ru/query_language/system.md (hunk 3 above)
Comment (BayoNet) [reply docs/ru/query_language/system.md]: It doesn't described yet, but I have a ticket.

File: docs/en/query_language/system.md (hunk 5 above)
Comment (BayoNet) [reply docs/en/query_language/system.md]: I have tried to send this query several times in a row, and there was no any effect, no exception nor something else. I don't know what to write.

Comment (blinkov) [reply docs/en/query_language/system.md]: Has no effect if distributed sends have already been stopped?

Comment (BayoNet) [reply docs/en/query_language/system.md]:
```
$ clickhouse-client 
ClickHouse client version 19.11.0.826.
Connecting to localhost:9000 as user default.
Connected to ClickHouse server version 19.11.0 revision 54423.

bayonet-VirtualBox :) system stop distributed sends

SYSTEM STOP DISTRIBUTED SENDS

Ok.

0 rows in set. Elapsed: 0.001 sec. 

bayonet-VirtualBox :) system stop distributed sends

SYSTEM STOP DISTRIBUTED SENDS

Ok.

0 rows in set. Elapsed: 0.001 sec. 
```

**Pull request merged**